stablehlo/dialect/StablehloOps.cpp

```diff
@@ -1330,19 +1330,26 @@ LogicalResult OutfeedOp::inferReturnTypes(
 
 LogicalResult SendOp::inferReturnTypes(
     MLIRContext* context, std::optional<Location> location, ValueRange operands,
-    DictionaryAttr, OpaqueProperties, RegionRange,
+    DictionaryAttr attributes, OpaqueProperties properties, RegionRange regions,
     SmallVectorImpl<Type>& inferredReturnTypes) {
+  SendOp::Adaptor adaptor(operands, attributes, properties, regions);
+  bool isDeviceToDevice = adaptor.getChannelHandle().getType() == 1;
```

Comment:
I think the PR description is okay. The enum type part is covered by ODS, and the value is checked in the verifier and noted as such in the description.

Reply:
> The enum type part is covered by ODS

Only that the value is a signed integer is covered by ODS, i.e. `type=-1` is valid under ODS, but not a valid enum value (since there is no enum).

`let parameters = (ins "int64_t":$handle, "int64_t":$type);`

Saying covered by C1 is also how the implementation works, in that invalid integer values ("enums") are only caught by C1 and throw the same error as C1, this is different from other "covered by ODS" constraints which error in generated op verification code

Reply:
I see what you mean, updated the PR description and the comment in the code to reflect the change.